### PR TITLE
Add an example `ControllerConfig` to sizing-guide

### PR DESCRIPTION
**Pull request by turkenf**
### Description of your changes

In this PR: Added an example of how to add configuration parameters for users.

Fixes: https://github.com/upbound/upjet/issues/189

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: docs/sizing-guide.md [hunk 1]
```diff
@@ -83,7 +83,40 @@ concurrent reconcilers to be used with each managed resource controller. Upjet
 executes certain Terrafom CLI commands asynchronously and this may result in 
 more than the `max-reconcile-rate` CLI invocations to be in flight at a given time.
 
+## Adding Configuration Parameters to ControllerConfig
+
+The following YAML can be used to add or update the above parameters. 
+Please run `kubectl apply -f` after copying the following into a local file.
```
Comment (jeanduplessis) [inline docs/sizing-guide.md:89]:
```suggestion
To apply the configuration parameters mentioned above, 
you need to add them as arguments to the `ControllerConfig`. 

In the example below you can see the options specified under the
`spec.args` section. 

You can apply this configuration by running `kubectl apply -f`
and referencing the local file path with the config.
```

File: docs/sizing-guide.md [hunk 2]
```diff
@@ -83,7 +83,40 @@ concurrent reconcilers to be used with each managed resource controller. Upjet
...
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws:v0.32.1
+  controllerConfigRef:
+    name: example-config
+```
+You will get the following warning, please ignore it:
```
Comment (jeanduplessis) [inline docs/sizing-guide.md:110]:
```suggestion
Note: Due to ControllerConfig being marked as deprecated, you will get the following warning, which you can ignore for now:
```

File: docs/sizing-guide.md [hunk 3]
```diff
@@ -83,7 +83,40 @@ concurrent reconcilers to be used with each managed resource controller. Upjet
...
+
+```
+Warning: This API is deprecated and is scheduled to be removed in a future release.
+```
+
+You can see more details for [ControllerConfig]
```
Comment (jeanduplessis) [inline docs/sizing-guide.md:116]:
```suggestion
Please take a look at the Crossplane documentation for more information about [ControllerConfig].
```

**Review by jeanduplessis — CHANGES_REQUESTED** [review]

File: docs/sizing-guide.md (hunk 3 above)
Comment (turkenf) [reply docs/sizing-guide.md]: Suggested changes have been made.

**Review by jeanduplessis — APPROVED** [review]

**Comment by jeanduplessis** [comment]
Thank you @turkenf looks good to me 🚀

**Pull request merged**